### ISSUE TITLE
 Implement Mock Server for IOT Timeseries API in NestJS

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#29]
## User story

1. As a developer
2. I want to implement a Mock Server for the IOT Timeseries API in NestJS,
3. So that the frontend can utilize it to retrieve pump data for both broken and working pumps without the need for a connection to insights hub.
see #27  for detailed description and examples. 

## Acceptance criteria
- [x] The Mock Server is implemented using NestJS.
-  [x] The server provides a GET method that accepts two parameters: entityid and propertyset.
- [x] The server replaces timestamps from the JSON file with current timestamps.
- [x] The server employs an algorithm to determine whether to return data for a broken pump or a working pump.
-  [x] The Mock Server can be accessed independently and does not require a connection to insights hub.

## Definition of done (DoD)

- [x] Added only after week 5
- [x] The same for all features
- [x]  Here goes the project specific part

## DoD general criteria

- [x] Feature has been fully implemented
- [x] Feature has been merged into the mainline
- [x] All acceptance criteria were met
- [x] Product owner approved features
- [x] All tests are passing
- [x] Developers agreed to release

**Comment by Sabo2k** [comment]
You can also use checkboxes for each acceptance using ```- [ ] Text``` criterium instead of code block 🙂

It then looks like this: 

- [ ] Accptance Criteria 1
